src/integrationTest/java/uk/gov/hmcts/reform/juddata/cameltest/testsupport/LeafIntegrationTestSupport.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.juddata.cameltest.testsupport;
+
+import static uk.gov.hmcts.reform.juddata.cameltest.testsupport.IntegrationTestSupport.setSourcePath;
+
+public interface LeafIntegrationTestSupport extends IntegrationTestSupport {
+
+    String[] file = {"classpath:sourceFiles/leaf/Roles.csv", "classpath:sourceFiles/leaf/Contracts.csv", "classpath:sourceFiles/leaf/Locations.csv", "classpath:sourceFiles/leaf/BaseLocations.csv"
+    };
+
+    String[] file_error = {"classpath:sourceFiles/leaf/Roles.csv", "classpath:sourceFiles/leaf/Contracts-invalid-header.csv", "classpath:sourceFiles/leaf/Locations.csv", "classpath:sourceFiles/leaf/BaseLocations.csv"
+    };
+
+    String[] file_jsr_error = {"classpath:sourceFiles/leaf/Roles_jsr.csv", "classpath:sourceFiles/leaf/Contracts_jsr.csv", "classpath:sourceFiles/leaf/Locations_jsr.csv", "classpath:sourceFiles/leaf/BaseLocations_jsr.csv"
+    };
+
```

Comment:
can we pick the file names from integration test yaml file so that it easy to change when it is used for LRD or SRD in future?

Reply:
These are more specific to JRD taste case. Will moved in JRD project only.